rizzino_a06c.js

```diff
@@ -0,0 +1,67 @@
+/* Julie Rizzino
+rizzino_a06c.js
+INFO 2124
+Thoendel
+1/23/20 */
+
+//winning numbers: 22, 42, 43, 47, 61;
+
+let lottoPicks = [];
+let powerBallPick = getLottoNumber(); 
+let winnings = 0; 
+
+/* WRITE YOUR SOLUTION BETWEEN THIS LINE */
+do {
+    lottoPicks.push(getLottoNumber());
+    //Pushes the getLottoNumber function to load a number into the lottoPicks array.
+} while (lottoPicks.length < 5);
+//Continues to get a number from the getLottoNumber function until the array has 5 numbers.
+
+for (let n = 0; n < lottoPicks.length; n++)
+//for loop iterates through the length of the lottoPicks array.
+{
+    console.log(lottoPicks[n]);
+    //I included a console.log of the lottoPicks array because I was checking if winning numbers were getting matched using the switch structure.
+    switch (n) {
```

Comment:
here you are matching based on the value of n - your counter - which will have a value of 0 to 4.
You want to match the value of the that item in your lottoPicks array. You output properly above!
So - if you had the following - everything should work:
switch(lottoPicks[n]) {...
if you had added a log for n - you would have see this - I'm sure. I like that you added output to try to debug! That is the way to do it.
-5 for switch +1 for debug